src/include/platform/ConfigurationManager.h

```diff
@@ -138,14 +139,14 @@ class ConfigurationManager
 
     virtual void LogDeviceConfig() = 0;
 
-    virtual bool IsCommissionableDeviceTypeEnabled()                              = 0;
-    virtual CHIP_ERROR GetDeviceTypeId(uint32_t & deviceType)                     = 0;
-    virtual bool IsCommissionableDeviceNameEnabled()                              = 0;
-    virtual CHIP_ERROR GetCommissionableDeviceName(char * buf, size_t bufSize)    = 0;
-    virtual CHIP_ERROR GetInitialPairingHint(uint16_t & pairingHint)              = 0;
-    virtual CHIP_ERROR GetInitialPairingInstruction(char * buf, size_t bufSize)   = 0;
-    virtual CHIP_ERROR GetSecondaryPairingHint(uint16_t & pairingHint)            = 0;
-    virtual CHIP_ERROR GetSecondaryPairingInstruction(char * buf, size_t bufSize) = 0;
+    virtual bool IsCommissionableDeviceTypeEnabled()                                 = 0;
+    virtual CHIP_ERROR GetDeviceTypeId(uint32_t & deviceType)                        = 0;
+    virtual bool IsCommissionableDeviceNameEnabled()                                 = 0;
+    virtual CHIP_ERROR GetCommissionableDeviceName(MutableCharSpan & deviceNameSpan) = 0;
```

Comment:
Please document that the callee will resize the span to the actual size of the character data.

Reply:
done